### PR TITLE
[ServiceNow v2] Add the add_as_entry arg to get-ticket-notes command

### PR DESCRIPTION
**Pull request by mmhw**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-34490)

## Description
Added the *add_as_entry* argument to the *servicenow-get-ticket-notes* command to optionally add the notes and work_notes as an entry to the War Room.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by DeanArbel — APPROVED** [review]
Looks great, good job. As part of the documentation effort, don't forget to add it to the readme.

File: Packs/ServiceNow/ReleaseNotes/2_5_58.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### ServiceNow v2
+
+- Added the *add_as_entry* argument to the *servicenow-get-ticket-notes* command to optionally add the notes and work_notes as an entry to the War Room.
```
Comment (julieschwartz18) [inline Packs/ServiceNow/ReleaseNotes/2_5_58.md:6]: Recommend:
Added the *add_as_entry* argument to the ***servicenow-get-ticket-notes*** command to optionally add ticket notes and work notes as notes in the War Room.

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml [hunk 2]
```diff
@@ -1317,6 +1317,13 @@ script:
       - sc_task
       - sc_req_item
       - sn_si_incident
+    - auto: PREDEFINED
+      defaultValue: false
+      description: Whether to add the ticket notes and work notes as an entry note in the War Room.
```
Comment (julieschwartz18) [inline Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml:1322]: Recommend:
Whether to add ticket notes and work notes as notes in the War Room.

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md [hunk 3]
```diff
@@ -1905,23 +1905,24 @@ Gets table names by a label to use in commands.
...
-| limit  | Maximum number of ticket notes. Default is 10.     | Optional | 
-| offset | Offset of the ticket notes.    | Optional |
-| use_display_value | Whether to retrieve notes using display values from ServiceNow or by accessing the sys_journal_field table. Defaults to the value set in the Cortex XSOAR instance configuration. | Optional |
-| ticket_type   | The type of the ticket for which notes should be retrieved when using the `use_display_value` option. Defaults to the ticket type set in the Cortex XSOAR instance configuration. | Optional |
+#### Input
 
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| id | Ticket System ID. | Required | 
+| limit | Maximum number of ticket notes. Default is 10. Default is 10. | Optional |
```
Comment (julieschwartz18) [inline Packs/ServiceNow/Integrations/ServiceNowv2/README.md:1921]: Recommend: remove the extra "Default is 10."

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md [hunk 4]
```diff
@@ -1905,23 +1905,24 @@ Gets table names by a label to use in commands.
...
-| ticket_type   | The type of the ticket for which notes should be retrieved when using the `use_display_value` option. Defaults to the ticket type set in the Cortex XSOAR instance configuration. | Optional |
+#### Input
 
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| id | Ticket System ID. | Required | 
+| limit | Maximum number of ticket notes. Default is 10. Default is 10. | Optional | 
+| offset | Offset of the ticket notes. Default is 0. | Optional | 
+| use_display_value | Whether to use `sysparm_display_value` to retrieve comments and work notes. Overrides the value set in the instance configuration. Possible values are: true, false. | Optional | 
+| ticket_type | The type of the ticket for which notes should be retrieved when using the `use_display_value` option. Possible values are: incident, problem, change_request, sc_request, sc_task, sc_req_item, sn_si_incident. Default is incident. | Optional |
```
Comment (julieschwartz18) [inline Packs/ServiceNow/Integrations/ServiceNowv2/README.md:1924]: Recommend:
The ticket type that notes should be retrieved for when using the `use_display_value` option. Possible values are: incident, problem, change_request, sc_request, sc_task, sc_req_item, sn_si_incident. Default is incident.

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md [hunk 5]
```diff
@@ -1905,23 +1905,24 @@ Gets table names by a label to use in commands.
...
+#### Input
 
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| id | Ticket System ID. | Required | 
+| limit | Maximum number of ticket notes. Default is 10. Default is 10. | Optional | 
+| offset | Offset of the ticket notes. Default is 0. | Optional | 
+| use_display_value | Whether to use `sysparm_display_value` to retrieve comments and work notes. Overrides the value set in the instance configuration. Possible values are: true, false. | Optional | 
+| ticket_type | The type of the ticket for which notes should be retrieved when using the `use_display_value` option. Possible values are: incident, problem, change_request, sc_request, sc_task, sc_req_item, sn_si_incident. Default is incident. | Optional | 
+| add_as_entry | Whether to add the ticket notes and work notes as an entry note in the War Room. Possible values are: true, false. | Optional |
```
Comment (julieschwartz18) [inline Packs/ServiceNow/Integrations/ServiceNowv2/README.md:1925]: Recommend:
Whether to add ticket notes and work notes as notes in the War Room. Possible values are true, false.

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md [hunk 6]
```diff
@@ -1905,23 +1905,24 @@ Gets table names by a label to use in commands.
 
 
 ### servicenow-get-ticket-notes
-***
-Gets notes from the specified ServiceNow ticket. Can be used by providing "Read permissions" for the sys_journal_field table, or by setting use_display_value=true.
 
+***
+Gets notes from the specified ServiceNow ticket. Notes can be retrieved either by granting "Read permissions" for the sys_journal_field table, or by setting the `use_display_value` parameter to true.
```
Comment (julieschwartz18) [inline Packs/ServiceNow/Integrations/ServiceNowv2/README.md:1910]: Recommend:
Gets notes from the specified ServiceNow ticket. Notes can be retrieved either by granting Read permissions for the sys_journal_field table, or by setting the `use_display_value` parameter to true.

**Review by julieschwartz18 — CHANGES_REQUESTED** [review]
Doc review complete. Note that I only reviewed the updates

**Pull request merged**